README.md

```diff
@@ -11,75 +11,108 @@ Generative AI is fallible in ways similar to people. It makes typos, misintepret
+
+3. **Add Bot as Collaborator**:
+    - Navigate to your GitHub repository
     - Go to Settings → Collaborators
     - Click "Add people"
-    - Search for your bot account via whatever method you prefer and select it
-    - Click "Add to repository"
+    - Search for your bot account and add it to the repository
     - Switch to your bot account to accept the invite
 
-[^1]: There is currently no way to generate fine-grained access tokens for collaborator access to repositories owned by
-individuals. When you give a classic Personal Access Token to the bot, you should assume from that point on that it can
-and will go rogue and attempt to abuse the broad permissions of that access token. As a repository owner, use
-collaborator permission settings and protected branches to restrict the bot's permissions to only the minimum required
-to perform its intended functions.
+4. **Get Anthropic API Key**:
+    - Sign up for an Anthropic account at https://console.anthropic.com
+    - Generate an API key from the console
+    - Ensure you have sufficient credits for API usage
+
+[^1]: There is currently no way to generate fine-grained access tokens for collaborator access to repositories owned by individuals. When you give a classic Personal Access Token to the bot, you should assume from that point on that it can and will go rogue and attempt to abuse the broad permissions of that access token. As a repository owner, use collaborator permission settings and protected branches to restrict the bot's permissions to only the minimum required to perform its intended functions.
+
+## Installation
+
+### Option 1: GitHub Action (Recommended)
+
+The easiest way to run the bot is as a GitHub Action that automatically responds to assigned issues and PR comments.
+
+#### Setup Instructions
+
+1. **Configure Repository Variables**:
+   - Go to your repository → Settings → Secrets and variables → Actions → Variables tab
+   - Add the following repository variables:
+     - `BOT_USERNAME`: Your bot's GitHub username
+     - `AUTHORIZED_USERNAME`: Your main GitHub username (who can trigger the bot)
 
-### Anthropic API Setup
+2. **Configure Repository Secrets**:
+   - Go to your repository → Settings → Secrets and variables → Actions → Secrets tab
+   - Add the following repository secrets:
+     - `BOT_GITHUB_TOKEN`: The Personal Access Token from your bot account
+     - `ANTHROPIC_API_KEY`: Your Anthropic API key
 
-1. Sign up for an Anthropic account at https://console.anthropic.com
-1. Generate an API key from the console
-1. Ensure you have sufficient credits for API usage
+3. **Add the GitHub Action Workflow**:
+   - Copy the [bot.yml workflow file](.github/workflows/bot.yml) from this repository to `.github/workflows/bot.yml` in your repository
+   - The workflow file contains all the necessary configuration for running the bot automatically on issue assignments and PR updates
 
-### Configuration
+### Option 2: Pre-built Binary
```

Comment:
Advantages: when run directly, the bot can handle tasks in multiple repositories. It also doesn't require any setup in the repositories to which the bot is contributing. Mention these advantages here